packages/shorebird_cli/test/src/commands/build/build_app_bundle_command_test.dart

```diff
@@ -112,12 +117,34 @@ void main() {
       );
 
       expect(result, equals(ExitCode.success.code));
-      expect(processExecutable, equals('flutter'));
-      expect(processArguments, equals(['build', 'appbundle', '--release']));
+      verify(
+        () => shorebirdProcess.run(
+          'flutter',
+          ['build', 'appbundle', '--release'],
+          runInShell: any(named: 'runInShell'),
```

Comment:
```suggestion
          runInShell: true,
```
do we want to also make sure it's run in shell?